### PR TITLE
README.md: update with new links to godoc and CI

### PR DESCRIPTION
**Pull request by ericchiang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # go-oidc
 
-[![GoDoc](https://godoc.org/github.com/coreos/go-oidc?status.svg)](https://godoc.org/github.com/coreos/go-oidc)
-[![Build Status](https://travis-ci.org/coreos/go-oidc.png?branch=master)](https://travis-ci.org/coreos/go-oidc)
+[![GoDoc](https://godoc.org/github.com/coreos/go-oidc?status.svg)](https://pkg.go.dev/github.com/coreos/go-oidc/v3/oidc)
```
Comment (mfridman) [inline README.md:3]: https://pkg.go.dev/github.com/coreos/go-oidc/v3/oidc returns a 404

Comment (ericchiang) [reply README.md:3]: Working on that :)

Comment (mfridman) [reply README.md:3]: Apologies.

Comment (ericchiang) [reply README.md:3]: Thanks for helping review! Should be up now. Having trouble with go mods, ha.

Comment (mfridman) [reply README.md:3]: Happy to help.

Once in a while you come across an oss project that has a great API, maintained and works as expected. Such projects illicit a feeling analogous to a neighbours cute puppy .. not really yours but can come play anytime and have a blast.

**Pull request merged**